document/cloud-client/analyze_form.py

```diff
@@ -0,0 +1,79 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# [START documentai_analyze_form]
+from google.cloud import documentai_v1beta2 as documentai
+
+def analyze_form(project_id='invoice-parser0', input_uri='gs://cloud-samples-data/documentai/form.pdf'):
```

Comment:
I'm sorta surprised the lint check isn't upset about this line.
Mind making this a multi-line thing? I believe the style check is supposed to be 88. 
@leahecole, did something change on that?

Reply:
It looks like this has been addressed for all samples. Please verify.